index.js

```diff
@@ -2,17 +2,58 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
+const list = document.querySelector('#animals')
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = 'orange'
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const img = document.querySelector('img');
+const h2 = document.querySelector('h2');
+const em = document.querySelector('em');
+const allAnimalSightings = document.querySelector('#animals');
```

Comment:
this seems to be a duplicate of line 4. Good descriptive variable name here, though!